certbot/compat/misc.py

```diff
@@ -2,42 +2,31 @@
 This compat module handles various platform specific calls that do not fall into one
 particular category.
 """
-import ctypes
-import errno
+from __future__ import absolute_import
+
+import os  # pylint: disable=os-module-forbidden
 import select
 import stat
 import sys
 
-from certbot import errors
-from certbot.compat import os
+try:
+    from win32com.shell import shell as shellwin32  # pylint: disable=import-error
+except ImportError:  # pragma: no cover
+    shellwin32 = None  # type: ignore
 
-UNPRIVILEGED_SUBCOMMANDS_ALLOWED = [
-    'certificates', 'enhance', 'revoke', 'delete',
-    'register', 'unregister', 'config_changes', 'plugins']
```

Comment:
This is probably fine, but what is the motivation of removing this list?

Reply:
This because on top of the symlinks that required already administrative privileges, several operations in `filesystem` will require them also, making effectively almost all operations in certbot unusable with them.

That, added to the fact that Certbot usually requires root on Linux, as stated by the documentation, that communicating with IIS is also an administrative operation, and that Windows implements metadata to programmatically require administrative privileges to run marked executable, make this distinction between certbot commands useless.

Reply:
That makes sense. I think some commands like `plugins` could still work without administrative privileges, but we can keep things simple and require it across the board until there's a good reason for us to do something differently.